src/model_parser.cc

```diff
@@ -293,13 +293,20 @@ ModelParser::InitOpenAI(
   response_output->second.datatype_ = "JSON";
   response_output->second.shape_.push_back(1);
 
+  // OpenAI in fixed schedule mode takes a timestamp input
+  if (inference_load_mode == InferenceLoadMode::FixedSchedule) {
+    auto delay_input = inputs_->emplace("timestamp", ModelTensor()).first;
+    delay_input->second.name_ = "timestamp";
+    delay_input->second.datatype_ = "UINT64";
+    delay_input->second.shape_.push_back(1);
+  }
```

Comment:
As mentioned, the new `--fixed-schedule` mode looks in the input data JSON for `timestamp` key to find timestamps.

We have to add the `timestamp` as an expected input in the ModelParser so that the DataLoader can read it in without errors.